ods_ci/utils/scripts/ocm/ocm.py

```diff
@@ -18,9 +18,16 @@
 dir_path = os.path.dirname(os.path.abspath(__file__))
 sys.path.append(dir_path + "/../")
 from logger import log
+
 # fmt: off
```

Comment:
does this comment have any sense here now?

Reply:
I don't insist on changing it now. It just doesn't seem to be relevant anymore since this commit changed the formatting for this part of code :slightly_smiling_face:

Anyway, feel free to merge it as it is.

Reply:
> I'm going to add a fixup commit removing the #fmt in the PR rn; let's not waste time on this small issue

Too late, adding the fixup dismissed the previous approvals ;(

I'm trying to remember if I already had perms to merge myself... I think not, but I am not sure, can't recollect precisely

Reply:
Did not have permissions. I still don't.